debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -214,6 +215,34 @@ public void shouldProduceEventsWithInitialSnapshot() throws Exception {
         assertRecordsAfterInsert(2, 3, 3);
     }
 
+    @Test
+    @FixFor("DBZ-1174")
+    public void shouldUseMicrosecondsForTransactionCommitTime() throws InterruptedException {
+        TestHelper.execute(SETUP_TABLES_STMT);
+        start(PostgresConnector.class, TestHelper.defaultConfig().build());
+        assertConnectorIsRunning();
+
+        // check records from snapshot
+        Instant inst = Instant.now();
+        // Microseconds since epoch, may overflow
+        final long microsSnapshot = TimeUnit.SECONDS.toMicros(inst.getEpochSecond()) + TimeUnit.NANOSECONDS.toMicros(inst.getNano());
```

Comment:
Replace with `Conversions.toEpochMicros()`.

Reply:
Ah yes, good spot. Added that method only later on. I'll update.